### ISSUE TITLE
[Component name] Short description of issue

### ISSUE THREAD

**Issue by Mika-89** [material-components/material-components-android#2987]
**Description:**
onSurfaceColor (background) is not equal for date and timepicker.
As you can see on the pictures the background value is different on date and timepicker to each other.
It should be the same color!

Second bug is the left and right margin. As you can see its smaller on datepicker as on timepicker.

**Expected behavior:**
The background color tint should be the same for date and/or timepicker

**Source code:**
I did not apply any theme changes to the default ones. App theme uses M3

**Minimal sample app repro:** 
![d1](https://user-images.githubusercontent.com/29900947/191434991-d0115c55-8317-457e-9edb-5d73364cdc89.png)

![t1](https://user-images.githubusercontent.com/29900947/191435006-7df81afc-32f3-4d73-abcf-a7236ef892cb.png)

**Android API version:**
Android 12

**Material Library version:**
com.google.android.material:material:1.8.0-alpha01

**Device:** 
Every device

To help us triage faster, please check to make sure you are using the [latest version](https://github.com/material-components/material-components-android/releases) of the library.

We also happily accept [pull requests](https://github.com/material-components/material-components-android/pulls).

**Comment by pekingme** [comment]
I'm seeing the same container background color between datepicker and timepicker with the latest snapshot. Could you verify?

**Comment by Mika-89** [comment]
No. It's also different like you can see in the screenshots with the latest snapshot

**Comment by drchen** [comment]
Thanks for catching this! I think that's a bug of date picker which should use surface3 instead of surface1 according to the spec: https://m3.material.io/components/date-pickers/specs

**Comment by drchen** [comment]
Paul can you help update that?

**Comment by Mika-89** [comment]
Please look at the start and end margin differences too.
The margin on date picker is smaller then the timepicker margin.
